packages/config/config/devices/0x031e/lzw30.json

```diff
@@ -123,6 +123,10 @@
 				{
 					"label": "Pink",
 					"value": 234
+				},
+				{
```

Comment:
Just a couple of pointers that will likely be requested.
where a param/option only applies to a certain firmware, use the "$if" property, before the other bits of info.

```
{
  "$if": "firmwareVersion >= 1.19",
  "label": "White",
  "value": 255
}
```

Reply:
Thanks for the pointer! I went ahead and made the change. I was looking at other config files instead of the documentation for guidance. If this gets accepted I'll see what I can do to clean up config files for other inovelli devices that I have.

Reply:
@blhoward2, will likely have something on his radar. So I would check in with him before going full throttle.